### PR TITLE
Completed a majority of Recommended section

### PR DESCRIPTION
**Pull request by timenglesf**
Started work on the outfit section, which for now is just a copy of the recommended folder, with the class names changed.
Commented out the console logs. Might be a good idea to  comment those out before pushing/creating a pull request, and then turning them back on on our local branches.

### CODE REVIEW

File: src/components/ItemsComponent/OutfitItems/OutfitItems.css
Comment (timenglesf) [inline src/components/ItemsComponent/OutfitItems/OutfitItems.css:1]: This is a copy with renamed classes. Not sure if  you need to read through the whole thing.

File: src/components/ItemsComponent/OutfitItems/OutfitItems.jsx
Comment (timenglesf) [inline src/components/ItemsComponent/OutfitItems/OutfitItems.jsx:1]: This is a copy with renamed classes. Not sure if  you need to read through the whole thing.

File: src/components/ItemsComponent/RecommendedItems/Card/Card.css
Comment (timenglesf) [inline src/components/ItemsComponent/RecommendedItems/Card/Card.css:1]: This is a copy with renamed classes. Not sure if  you need to read through the whole thing.

File: src/components/ItemsComponent/RecommendedItems/Card/Card.jsx
Comment (timenglesf) [inline src/components/ItemsComponent/RecommendedItems/Card/Card.jsx:1]: This is a copy with renamed classes. Not sure if  you need to read through the whole thing.

**Review by L-ren — APPROVED** [review]
LGTM, thanks for commenting out the console logs

**Pull request merged**